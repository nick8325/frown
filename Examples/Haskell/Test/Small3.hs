f2 x = do {
x
}
