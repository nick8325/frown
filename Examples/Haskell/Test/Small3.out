module Main where
f2 x = do x
